libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-.quiz { 
-  min-height: 1000px;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+.quiz-page * {
+  box-sizing: border-box;
 }
 
-.quiz > div:not([class]) {
-  display:none;
+.quiz-page .feds-topnav-wrapper {
+  padding: 0 20px;
 }
 
-.quiz-container {
-  display: flex;
-  align-items: center;
-  justify-content: flex-start;
-  flex-direction: column;
-  color: var(--color-white);
-  width: 100%;
+div[data-valign="middle"] {
+  align-self: center;
 }
 
-.quiz-container .foreground {
-  position: relative;
-  margin: var(--spacing-m) 0 var(--spacing-s);
-  display: flex;
-  flex-direction: column;
-  gap: var(--spacing-m);
-  justify-content: center;
+.quiz-load {
   text-align: center;
 }
 
-.quiz-option {
-  display: flex;
-  align-items: stretch;
-  flex: 1;
-  flex-direction: row;
-  width: auto;
-  margin: 5rem 0;
-  justify-content: center;
-  overflow-y: auto;
-}
-
-.quiz-container.small .foreground {
-  padding: var(--spacing-xl) 0;
-}
-
-.quiz-container.large .foreground {
-  padding: var(--spacing-m) 0;
-  gap: 0;
-}
-
-.quiz-container.small,
-.quiz-container.quiet,
-.quiz-container.inline {
-  min-height: 360px;
-}
-
-.quiz-container.inline hr {
-  margin-top: var(--spacing-m);
-  margin-bottom: 0;
-}
-
-.quiz-container.light,
-.quiz-container.quiet,
-.quiz-container.inline {
-  color: var(--text-color);
+.quiz {
+  min-height: calc(100vh - 131px);
 }
 
-.quiz-container .action-area {
-  display: flex;
-  margin: 0;
-  gap: var(--spacing-s);
-  flex-wrap: wrap;
+.quiz-container {
   align-items: center;
+  color: var(--color-white);
+  display: flex;
+  flex-direction: column;
 }
 
-.quiz-container.centered {
-  text-align: center;
-}
-
-.quiz-container.centered .foreground,
-.quiz-container.centered .action-area {
-  justify-content: center;
-}
-
-.quiz-container .background.has-video {
-  bottom: 0;
-  left: 0;
-  overflow: hidden;
-  position: absolute;
-  right: 0;
-  top: 0;
+.quiz-background {
+  z-index: 0;
```

Comment:
Is this actually needed, since 0 should be similar to the default value?

Reply:
Nope! good catdh